### PR TITLE
add a message for windows in bundletool check

### PR DESCRIPTION
**Pull request by KazuCocoa**
According to https://github.com/appium/appium/issues/12269#issuecomment-486553116, node-which can find the module by adding '.JAR' in their PATHEXT..

### CODE REVIEW

File: lib/android.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ class OptionalAppBundleCheck extends DoctorCheck {
   }
 
   async fix () { // eslint-disable-line require-await
-    return 'bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it';
+    return 'bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it' +
+      `${system.isWindows() ? '. Please consider adding ".JAR" in your PATHEXT in order to find it' : ''}`;
```
Comment (mykola-mokhnach) [inline lib/android.js]: Also consider adding the ".jar" extension into your PATHEXT environment variable in order to fix the problem for Windows

File: lib/android.js [hunk 2]
```diff
@@ -71,7 +71,8 @@ class OptionalAppBundleCheck extends DoctorCheck {
 
   async fix () { // eslint-disable-line require-await
     return 'bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it' +
-      `${system.isWindows() ? '. Please consider adding ".JAR" in your PATHEXT in order to find it' : ''}`;
+      `${system.isWindows() ? '. Also consider adding the ".jar" extension into your PATHEXT environment variable in order to fix the problem for Windows' : ''}`;
+
```
Comment (mykola-mokhnach) [inline lib/android.js]: extra empty line here

**Comment by KazuCocoa** [comment]
@imurchie 
Could you release new ver when you have a chance?

For users, the release has below items:

- add mjpeg consumer check #64
- add a helpful message for bundletool jar file on windows environment (this PR)

**Comment by imurchie** [comment]
`appium-doctor@1.10.0` is out.

**Pull request merged**